src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_data_source/components/header/header.tsx

```diff
@@ -58,111 +100,109 @@ export const Header: React.FC<HeaderProps> = (props: HeaderProps) => {
-        id={'useDataSource'}
-        label={
+        <EuiText size="s" className="dataSourceRadioHelperText">
           <FormattedMessage
-            id="indexPatternManagement.createIndexPattern.stepDataSource.useDataSourceLabel"
-            defaultMessage="Choose data source"
-          />
-        }
-        checked={dataSourceChecked}
-        onChange={(e) => onChangeDataSourceChecked(e)}
-        compressed
-      />
-      <EuiText size="s" className="dataSourceRadioHelperText">
-        <FormattedMessage
-          id="indexPatternManagement.createIndexPattern.stepDataSource.useDataSource.helperText"
-          defaultMessage="Connect to an existing external data source."
-        />
-      </EuiText>
-      {dataSourceChecked && (
-        <EuiFlexItem grow={false}>
-          <EuiSpacer size="m" />
-          <SavedObjectFinderUi
-            key="searchSavedObjectFinder"
-            onChoose={onDataSourceSelected}
-            showFilter={false}
-            noItemsMessage={i18n.translate(
-              'indexPatternManagement.createIndexPattern.searchSelection.notFoundLabel',
-              {
-                defaultMessage: 'No data sources have been configured yet.',
-              }
-            )}
-            savedObjectMetaData={[
-              {
-                type: 'data-source',
-                getIconForSavedObject: () => 'apps', // todo: #2034
-                name: i18n.translate(
-                  'indexPatternManagement.createIndexPattern.searchSelection.savedObjectType.dataSource',
-                  {
-                    defaultMessage: 'Data Source',
-                  }
-                ),
-              },
-            ]}
-            fixedPageSize={DATA_SOURCE_PAGE_SIZE}
-            uiSettings={uiSettings}
-            savedObjects={savedObjects}
+            id="indexPatternManagement.createIndexPattern.stepDataSource.useDataSource.helperText"
+            defaultMessage="Connect to an existing external data source."
           />
-        </EuiFlexItem>
-      )}
-      <EuiSpacer size="m" />
-      <EuiFlexGroup justifyContent="flexEnd">
-        <EuiFlexItem grow={false}>
-          <EuiButton
-            data-test-subj="createIndexPatternStepDataSourceNextStepButton"
-            fill
-            iconSide="right"
-            iconType="arrowRight"
-            onClick={() => goToNextStep(dataSourceRef)}
-            isDisabled={isNextStepDisabled && !defaultChecked}
-          >
-            <FormattedMessage
-              id="indexPatternManagement.createIndexPattern.step.nextStepButton"
-              defaultMessage="Next step"
-            />
-          </EuiButton>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </div>
+        </EuiText>
+        {dataSourceChecked && (
+          <EuiFlexItem grow={false}>
+            <EuiSpacer size="m" />
+            <EuiSelectable
+              aria-label="Search data sources"
```

Comment:
I would have preferred this to use i18n but won't block the PR on it.

Reply:
will add